### PR TITLE
Re-introduce config filtering on daemon side

### PR DESCRIPTION
**Pull request by dmzmk**

### CODE REVIEW

File: myceliald/src/main.rs [hunk 1]
```diff
@@ -270,7 +270,7 @@ impl Daemon {
             _ => Reset::None,
         };
         if let Reset::State = reset_type {
-            tracing::info!("resetting state");
+            tracing::warn!("resetting state");
```
Comment (wwkeyboard) [inline myceliald/src/main.rs:273]: This should be a more descriptive warning considering what it's going to the client.

**Review by wwkeyboard — APPROVED** [review]
Just some non-blocking questions.

File: myceliald/src/http_client.rs [hunk 2]
```diff
@@ -135,6 +130,21 @@ impl HttpClient {
...
+                    .into_iter()
+                    .filter(|pipe| match TryInto::<Config>::try_into(pipe.clone()) {
+                        Err(e) => {
+                            tracing::error!("invalid pipe config: {e}");
+                            false
+                        }
+                        Ok(config) => config
+                            .get_sections()
+                            .iter()
+                            .any(|section| section.get("client") == this_daemon.as_ref()),
```
[the remark this answers is not in the document]
Comment (dmzmk) [reply myceliald/src/http_client.rs:145]: apart from running pipes, which shouldn't be running on current daemon - none

File: myceliald/src/main.rs (hunk 1 above)
Comment (dmzmk) [reply myceliald/src/main.rs:273]: Agree, but will address this issue in different PR

**Pull request merged**